### DISCUSSION TITLE
Unable to find docfx.json in the pre-release version 3.00

### DISCUSSION THREAD

**Discussion by Cally0611** [dotnet/docfx#10214] [Q&A]
Hi all,
I am working with the docfx and the one in our production uses the pre-release version - 3.0.0. I am not able to find the docfx.json, there is a docfx.yml and according to repository's readme file, there is no command like docfx docfx.json, the command used earlier was docfx build. However, in the latest version 2.77.0, there is a requirement to use the docfx.json as config file. I feel like I am missing out something. How was it done in the earlier versions ? The code in the repository was last updated in 2022.

**Comment by lanthonyneville** [comment]
The v3 version is no longer being developed ([see here](https://github.com/dotnet/docfx/discussions/8277#discussioncomment-4409645)). The v2 version is the one being developed with 2.77.0 being the latest version.
